### DISCUSSION TITLE
Add "copy metadata" feature to duplicate checker

### DISCUSSION THREAD

**Discussion by sevengovern** [stashapp/stash#2876] [Ideas]
When deleting duplicates, sometimes the lower-quality version of the video has already been organized/labeled with metadata. It would be great if there were a way to copy the metadata from one file to another within the duplicate checker interface before deleting the duplicate.

**Comment by DogmaDragon** [comment]
Dupe of this https://github.com/stashapp/stash/issues/1088

**Reply by sevengovern** [reply]
Good call, I searched "duplicate" and for some reason that didn't come up.

**Reply by scruffynerf** [reply]
hopefully post file refactor, this will be easier.

In the short term, I wrote this: https://github.com/scruffynerf/stashcopyscenemetadata

**Reply by scruffynerf** [reply]
also https://github.com/stashapp/CommunityScripts/tree/main/plugins/phashDuplicateTagger
is better than the built in Duplicate checker in some ways, since it tags items, and you can then sort/filter as needed.
